Marlin/Marlin_main.cpp

```diff
@@ -1491,13 +1491,6 @@ static void setup_for_endstop_move() {
     run_z_probe();
     float measured_z = current_position[Z_AXIS];
 
-    #if Z_RAISE_BETWEEN_PROBINGS > 0
-      if (probe_action == ProbeStay) {
-        do_blocking_move_to(current_position[X_AXIS], current_position[Y_AXIS], current_position[Z_AXIS] + Z_RAISE_BETWEEN_PROBINGS); // this also updates current_position
-        st_synchronize();
-      }
-    #endif
-
```

Comment:
Good PR, but you may want to redo it, leaving out the portion deleting these lines. I have another PR #2281 for this specific issue already in the queue, and you may get a conflict after it is merged. But maybe not… you could also wait till the merge is done and then see. Just waiting for the PR to get some review…